recipes-appfw/example-app-node/files/manifest

```diff
@@ -9,10 +9,9 @@
     },
     "command": [  "/usr/bin/node", "/lib/node_modules/nodetest/example.js" ],
     "autostart": "yes",
-     "container": {
+    "container": {
         "type": "nspawn",
         "network": "VirtualEthernet",
-        "sharedsystem": false,
-        "portmap": 12345,
+        "portmap": [ { "proto": "tcp", "host": 12345, "container": 12345 } ],
```

Comment:
often it is good to use different container and host numbers so you know the code works, sometimes it "happens to work" since you may use the wrong variable but the value is right. Then use something like "container": 80.